metrontagger/options.py

```diff
@@ -42,7 +42,12 @@ def make_parser() -> argparse.ArgumentParser:
         action="store_true",
         default=False,
     )
-    parser.add_argument("--id", help="Identify file for tagging with the Metron Issue Id.")
+    parser.add_argument(
+        "--id",
+        help="Identify file for tagging with the Metron Issue Id, or restrict directory matches to issues from a specific Metron Series Id.",
+        type=int,
```

Comment:
I added this type requirement to `--id`, even though it didn't used to be there, just to emphasize that the id (whether series or issue) needs to be the numeric metron id, not the slug from the URL or something. When I was first using metron-tagger, I was trying things like `--id tales-of-suspense-1959-1` and wondering why it wasn't working. This will make it give a nice readable error message:

```
metron-tagger: error: argument --id: invalid int value: 'tales-of-suspense-1959-1'
```